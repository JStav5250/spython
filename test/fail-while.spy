def foo()->int:
	if true:
		return 3
	else:
		return 4

print(foo())

while foo():
	print(3)
