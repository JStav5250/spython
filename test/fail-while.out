TypeError: invalid boolean type in 'while' statement (found int but expected bool)!!
